### PR TITLE
PMM-6876 operators in response

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-6876](https://jira.percona.com/browse/PMM-6876)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1302

- [ ] Links to other linked pull requests (optional).

- https://github.com/percona/pmm/pull/637
- https://github.com/percona-platform/dbaas-controller/pull/106
- https://github.com/percona-platform/dbaas-api/pull/77
- https://github.com/percona/pmm-managed/pull/573

---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

File: services/management/dbaas/kubernetes_server.go [hunk 1]
```diff
@@ -52,19 +53,46 @@ func (k kubernetesServer) ListKubernetesClusters(ctx context.Context, _ *dbaasv1
...
+					Xtradb: new(dbaasv1beta1.Operator),
+					Psmdb:  new(dbaasv1beta1.Operator),
+				},
+			}
+			resp, e := k.dbaasClient.CheckKubernetesClusterConnection(ctx, cluster.KubeConfig)
+			if e != nil {
+				clusters[i].Status = dbaasv1beta1.KubernetesClusterStatus_KUBERNETES_CLUSTER_STATUS_UNAVAILABLE
+			}
+
+			if resp == nil {
```
Comment (askomorokhov) [inline services/management/dbaas/kubernetes_server.go:76]: Looks like this case only possible with error https://github.com/percona-platform/dbaas-controller/pull/106/files#diff-611c99343ad28b77556d6ca5ebfe5d6b63a2f6e4cbaf38927a605018c220351dR50
 
so may be we can do:

```
if e != nil {
    clusters[i].Status = dbaasv1beta1.KubernetesClusterStatus_KUBERNETES_CLUSTER_STATUS_UNAVAILABLE
    return 
}
```

**Review by askomorokhov — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by askomorokhov** [comment]
@BupycHuk , CI failed.

**Comment by BupycHuk** [comment]
@askomorokhov fixed

**Comment by AlekSi** [comment]
@BupycHuk A lot of merge conflicts now

**Pull request merged**